### PR TITLE
Fix | Fix API Description for EncryptColumnEncryptionKey Paramter

### PR DESCRIPTION
**Pull request by karinazhou**

### CODE REVIEW

**Comment by johnnypham** [comment]
A bit late for this but "Plain text" should also be one word in `SqlColumnEncryptionAzureKeyVaultProvider.EncryptColumnEncryptionKey`.

File: src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/SqlColumnEncryptionAzureKeyVaultProvider.cs [hunk 1]
```diff
@@ -213,7 +213,7 @@ public override byte[] DecryptColumnEncryptionKey(string masterKeyPath, string e
         /// </summary>
         /// <param name="masterKeyPath">Complete path of an asymmetric key in Azure Key Vault</param>
         /// <param name="encryptionAlgorithm">Asymmetric Key Encryption Algorithm</param>
-        /// <param name="columnEncryptionKey">Plain text column encryption key</param>
+        /// <param name="columnEncryptionKey">Plaintext column encryption key</param>
```
Comment (cheenamalhotra) [inline src/Microsoft.Data.SqlClient/add-ons/AzureKeyVaultProvider/SqlColumnEncryptionAzureKeyVaultProvider.cs:216]: Nit:
```suggestion
        /// <param name="columnEncryptionKey">The plaintext column encryption key.</param>
```

File: doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCertificateStoreProvider.xml [hunk 2]
```diff
@@ -26,7 +26,7 @@
             <param name="masterKeyPath">The master key path.</param>
             <param name="encryptionAlgorithm">The encryption algorithm. Currently, the only valid value is: <c>RSA_OAEP</c>
             </param>
-            <param name="columnEncryptionKey">The encrypted column encryption key.</param>
+            <param name="columnEncryptionKey">The plaintext column encryption key.</param>
```
Comment (David-Engel) [inline doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCertificateStoreProvider.xml:29]: When I first read the old line, I interpreted "encrypted" to apply to the column. In other words, "The encryption key for the encrypted column". If we are clarifying that the key is plaintext, perhaps we could say:
```suggestion
            <param name="columnEncryptionKey">The plaintext encryption key for the encrypted column.</param>
```

How do the ODBC and JDBC drivers phrase it?

Comment (David-Engel) [reply doc/snippets/Microsoft.Data.SqlClient/SqlColumnEncryptionCertificateStoreProvider.xml:29]: Looking at the other driver docs, I rescind my suggestion. The current seems appropriate in the context of the other methods and other driver descriptions.

**Review by David-Engel — APPROVED** [review]

**Review by cheenamalhotra — APPROVED** [review]

**Pull request merged**